### PR TITLE
updating mailroom per Natasha's comments

### PR DESCRIPTION
**Pull request by esokada**
Thanks for your help! I believe I have addressed all your comments.

### CODE REVIEW

File: students/esokada/lesson03/mailroom.py [hunk 1]
```diff
@@ -15,24 +15,22 @@
...
-    if donationamount == "cancel":
+    if donationamount == "C":
         return
-    existingdonor = False
     for d in donors:
         if d[0] == donorname:
-            existingdonor = True
             break
-    if existingdonor == False:
+    else:
```
Comment (natasha-aleksandrova) [inline students/esokada/lesson03/mailroom.py]: this is close.. but here is a more condensed version:
```
    while True:
        donorname = input("Please input a name, "list" command or "C" to cancel:\n")  # clarify what a user can input
        if donorname == "C":
            return
        elif donorname == "list":
            for donor in donors:
                print(donor[0])
        else:
            # here we will be adding donors
            donation_amount = input("Please enter a donation amount:\n")
            for donor in donors:
                if donor[0] == donorname:
                    donor[1].append(donation_amount)
            else:
                donors.append([donorname, [donation_amount]])

            print(f"Dear {donorname}, thank you for your generous donation of ${float(donation_amount):.2f}.\n")
```

Comment (natasha-aleksandrova) [reply students/esokada/lesson03/mailroom.py]: please review above ^^ let me know if you have questions.

Comment (esokada) [reply students/esokada/lesson03/mailroom.py]: Thanks, that makes sense!  I will incorporate these changes into next mailroom assignment.

Comment (natasha-aleksandrova) [reply students/esokada/lesson03/mailroom.py]: sounds good!

**Pull request merged**